### ISSUE TITLE
Unknown error using cssnano

### ISSUE THREAD

**Issue by Shoplifter** [postcss/postcss#1521]
when running following command (cli), `postcss web/src/*.css -u cssnano -d web/min --no-map `I get this errror:
````
Unknown error from PostCSS plugin. Your current PostCSS version is 8.2.4, but cssnano uses 7.0.35. Perhaps this is the source of the error below.
Error [BrowserslistError]: Unknown browser query `basedir=$(dirname "$(echo "$0" | sed -e 's`. Maybe you are using old Browserslist or made typo in query.
    at unknownQuery (C:\Users\XY\AppData\Roaming\npm\node_modules\cssnano\node_modules\browserslist\index.js:285:10)
    at C:\Users\XY\AppData\Roaming\npm\node_modules\cssnano\node_modules\browserslist\index.js:372:11
    at Array.reduce (<anonymous>)
    at resolve (C:\Users\XY\AppData\Roaming\npm\node_modules\cssnano\node_modules\browserslist\index.js:318:18)
    at browserslist (C:\Users\XY\AppData\Roaming\npm\node_modules\cssnano\node_modules\browserslist\index.js:444:21)
    at C:\Users\XY\AppData\Roaming\npm\node_modules\cssnano\node_modules\postcss-reduce-initial\dist\index.js:34:53
    at initializePlugin (C:\XY\baum\AppData\Roaming\npm\node_modules\cssnano\dist\index.js:31:51)
    at async LazyResult.runAsync (C:\Users\XY\AppData\Roaming\npm\node_modules\postcss\lib\lazy-result.js:358:11)
    at async Promise.all (index 2) {
  browserslist: true
}
````
I've been using this script for a long time, this happended only after the latest updates.

**Comment by ai** [comment]
You can some problems with Browserslist config. Check `.browserslistrc`, `browserslist` key in `package.json` and `BROWSERSLIST` environment variable.

**Comment by Shoplifter** [comment]
@ai I dont have any of` .browserslistrc` or `browserslist` in my `package.json`.
all required packages are installed globally.
only thing in the package.json ist configuration for the command metioned above.
Sorry, I am not very familiar with node / npm - dont know how to check env vars.
If I remember right, when I ran some postcss command, I was asked asked to ro run something like` npx .. browserlist ..`.
I did that and the warning dissappeared then, but now I got this error.
